app/src/main/java/com/example/nvdovin/weatherapp/presentation/main/weather/detail/DetailsView.java

```diff
@@ -0,0 +1,6 @@
+package com.example.nvdovin.weatherapp.presentation.main.weather.detail;
+
+public interface DetailsView {
+
```

Comment:
It is not used now, but methods will be added when i will implement US. Delete it anyway?